docs/ops/comparison/Equal_1.md

```diff
@@ -4,48 +4,50 @@
 
 **Category**: Comparison binary operation
 
-**Short description**: *Equal* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *Equal* performs element-wise comparison operation with two given input tensors applying multi-directional broadcast rules specified in the *auto_broadcast* attribute.
+
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and *auto_broadcast* attributes is not *none*. Broadcasting is performed according to *auto_broadcast* value.
+
+After broadcasting *Equal* does the following with the input tensors *a* and *b*:
+
+\f[
+o_{i} = a_{i} == b_{i}
+\f]
 
 **Attributes**:
 
 * *auto_broadcast*
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+    * *none* - no auto-broadcasting is allowed, all input shapes should match,
+    * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md),
+    * *pdpd* - PaddlePaddle-style implicit broadcasting, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
   * **Type**: string
   * **Default value**: "numpy"
   * **Required**: *no*
 
 **Inputs**
 
-* **1**: A tensor of type *T*. **Required.**
-* **2**: A tensor of type *T*. **Required.**
+* **1**: A tensor of type *T* and arbitrary shape. **Required.**
+* **2**: A tensor of type *T* and arbitrary shape. **Required.**
 
 **Outputs**
 
-* **1**: The result of element-wise comparison operation. A tensor of type boolean.
+* **1**: The result of element-wise **comparison** operation applied to the input tensors. A tensor of type **boolean** and the same shape equal to broadcasted shape of two inputs.
```

Comment:
What do you think about adding `boolean` to the **Types** section as \*T_OUT\* and use it here?

Reply:
Was it used somewhere else? I have never seen this kind of spec structure before.

Reply:
> As in [ReduceLogicalAnd](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/reduction/ReduceLogicalAnd_1.md) and [ReduceLogicalOr](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/reduction/ReduceLogicalOr_1.md):
> *T_BOOL*: `boolean`.

https://github.com/openvinotoolkit/openvino/pull/6852#discussion_r683295562